axelor-account/src/main/resources/views/AnalyticMoveLine.xml

```diff
@@ -242,11 +242,14 @@
 
   <action-attrs name="action-attrs-domain-onselect-analytic-account">
     <attribute name="domain" for="analyticAccount"
-      expr="eval: &quot;self.statusSelect = 1 AND self.analyticAxis.id = ${analyticAxis.id}  AND self.company.id = ${analyticAxis.company.id}&quot;"
+      expr="eval: &quot;self.statusSelect = 1 AND (self.analyticAxis.id = ${analyticAxis?.id} AND (self.company IS NULL OR self.company.id = ${analyticAxis?.company?.id}))&quot;"
```

Comment:
We don't want to get analyticAccounts with a null company, but here we'll get them with: self.company IS NULL
Please remove it